tests/func/test_init.py

```diff
@@ -1,53 +1,52 @@
 import logging
 import os
 
+import pytest
+
 from dvc.config import Config
 from dvc.exceptions import InitError
 from dvc.main import main
 from dvc.repo import Repo as DvcRepo
-from tests.basic_env import TestDir, TestGit
 
 
-class TestInit(TestGit):
-    def _test_init(self):
-        self.assertTrue(os.path.exists(DvcRepo.DVC_DIR))
-        self.assertTrue(os.path.isdir(DvcRepo.DVC_DIR))
+def _check_dirs():
+    assert os.path.exists(DvcRepo.DVC_DIR)
+    assert os.path.isdir(DvcRepo.DVC_DIR)
 
-    def test_api(self):
-        DvcRepo.init().close()
 
-        self._test_init()
+def test_api_init(scm):
+    DvcRepo.init().close()
+    _check_dirs()
```

Comment:
Just the check for `isdir` should be enough, so I think it'd be more clearer if we get rid of `_check_dirs` and replace them with `os.path.isdir(repo.DVC_DIR)`. 1 less indirection.